README

```diff
@@ -1,2 +1,10 @@
-# Neuromod: xxxx subdataset
+# CNeuromod: [xxxx subdataset](https://github.com/courtois-neuromod/xxxxx)
 
+This BIDS dataset is part of the [Courtois-Neuromod dataset](https://docs.cneuromod.ca/).
+
```

Comment:
I would add a sentence here on BIDS. 

OPTIONAL: 

This subdataset is organized following the [brain imaging data structure](https://bids.neuroimaging.io/).
OR
This subdataset is organized following principles which mimic the [brain imaging data structure](https://bids.neuroimaging.io/), but it does not conform to an established standard.